app/blocks/input/prefixScript.js

```diff
@@ -33,16 +33,25 @@ class PrefixScript extends InputBlock {
       return false
     }
     var regex = ['^']
-    if (!this.isScoped) {
+    if (this.isScoped) {
+      if (this.args.match(/^r/i)) {
```

Comment:
I'm totally in favor of code that is forgiving of user error. I'm just worried if the code is too obscure for future generations. I'm sure it's fine. I'll approve the PR and merge.